pyuvsim/uvsim.py

```diff
@@ -9,25 +9,27 @@
 the simulation.
 """
 
-import numpy as np
-import yaml
 import warnings
+
 import astropy.units as units
+import numpy as np
+import yaml
+from astropy.constants import c as speed_of_light
 from astropy.coordinates import EarthLocation
 from astropy.units import Quantity
-from astropy.constants import c as speed_of_light
 from pyuvdata import UVData
 
-from . import mpi
+try:
+    from . import mpi
+except ImportError:  # pragma: no cover
+    mpi = None
```

Comment:
and again here. should we be worried about the None case or is it covered further in?

Reply:
It's handled properly here.